docs/reference.cue

```diff
@@ -429,10 +429,52 @@ _values: {
 		default: #Timestamp | null
 	}
 
+	description: """
+		Vector supports a wide variety of timestamp formats. You can use:
+
+		1. One of the [built-in formats](#timestamp_formats) listed in the table below.
+		2. Custom timestamps using the [time format specifiers](\(urls.chrono_time_formats))
+			from Rust's `chrono` library. Custom timestamps need to be prefixed with
+			`timestamp|`, for example `timestamp|%Y-%m-%d %H:%M:%S`.
+		"""
```

Comment:
So, I should have clarified. I meant to move this specifically to the [`types`](https://github.com/timberio/vector/blob/b1d01894b46a55ad640bb676b547b2a061ec8c52/docs/reference/components.cue#L785) option. Vector's internal "timestamp" type doesn't support formats since it is represented as a native timestamp. The types are only relevant when parsing strings into timestamps.

Reply:
Ah, gotcha. Makes sense. I'm thinking we should expand this to not just explain timestamps but also list all of the types that we accept.

Reply:
@binarylogic Soooo, I don't love what I've done here. On the one hand I want this to be data driven, but I'm not sure how to accomplish that without expanding the `#Schema` to allow for arbitrary fields (which I've done here). Another option would be to provide raw Markdown tables, which actually may not be terrible given that this is unlikely to undergo significant change in the future.

Reply:
Actually, after thinking about a bit more, I'm thinking the raw Markdown tables approach is the way to go here given that this will soon only apply to non-latest Vector. It's less work and doesn't require us to bend `#Schema`. I can work that up quick if you want to go that way.